include/ultra64/gbi.h

```diff
@@ -1334,7 +1365,10 @@ typedef struct {
 } Hilite_t;
 
 typedef union {
-    Light_t l;
+    Light_t    l;
+#ifdef F3DEX_GBI_PL
+    PosLight_t p;
+#endif
```

Comment:
If you can share these existing community gbi.h implementations I can look at what they do, but my predisposition is that I'm not especially interested in using an old implementation when this implementation is already correct and does what we want.

Reply:
This is the community's pos light GBI for use with the SGI original GBI: https://github.com/HackerN64/HackerSM64/blob/master/include/n64/PR/gbi-poslight.h This is what I based the F3DEX3 pos light GBI on.

This is the community's reverse-engineered (non SGI) implementation: https://github.com/z64tools/z64hdr/blob/main/include/ultra64/gbi.h

Out of curiosity, I checked what the MM decomp uses (I don't think this existed when F3DEX3 was being started): https://github.com/zeldaret/mm/blob/main/include/PR/gbi.h They do have the union of the light type and the point light type called `Light`, but they also have the point light type named `PointLight_t` instead of `PosLight_t`.

If you think there is a good reason to switch away from the first community implementation already in use (for example, easier use of F3DEX3 in MM in the future), I'm open to it, but we should switch to some other standard community implementation, not make something up just for this repo.

Reply:
That first header in HackerSM64 is very suspicious. This comment `/* version 990705 */` suggests to me that this is probably a leaked header being appropriated as a community header. I personally have no wish to import this into HackerOoT.

The z64hdr gbi is glank's gbi with an added structure. The new geometry mode is altogether missing and the `LightPos_t` structure has poor field names. This implementation is simply not up to standard.

Of these choices, MM decomp's is far and away the best choice but it's still not as complete as this PR's. I'd rather upstream changes to MM's gbi.h than adopt any of these.